crowbar_framework/app/models/network_service.rb

```diff
@@ -428,13 +429,13 @@ def enable_interface(bc_instance, network, name)
     node.save
 
     Rails.logger.info("Network enable_interface: Assigned: #{name} #{network}")
-    [200, build_net_info(role, network)]
+    [200, build_net_info(role, network, nil)]
```

Comment:
@vuntz I might be missing something obvious by why is enable_interface passing `nil` here instead of the node?